src/dataverseClient.ts

```diff
@@ -173,6 +173,18 @@ export class DataverseClient {
     return this.postRequest(url, '')
   }
 
+  public async updateDataset(datasetId: string, datasetInformation: BasicDatasetInformation): Promise<AxiosResponse> {
+    const url = `${this.host}/api/datasets/:persistentId/versions/:draft?persistentId=${datasetId}`
```

Comment:
this method will only allow draft versions to be updated, is this correct?

Reply:
So so, if the dataset is in DRAFT state it will replace the old metadata, but if the dataset is in PUBLISHED state it will create a new version in DRAFT state.

Reply:
since this method is a general one, should we allow the user to specify the version? right now what I can see is that if we update through this method, we will always target to update the `draft` version of it. In your testing call this is not how it was done

Reply:
Makes sense, LGTM then 👏 🚀